### PR TITLE
Able to use Route as EXPOSE_SERVICE_TYPE in tests

### PR DESCRIPTION
**Pull request by dmvolod**

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]

File: pkg/launcher/launcher.go [hunk 1]
```diff
@@ -118,7 +118,7 @@ func Launch(params Parameters) {
 	}
 
 	// Setup Scheme for Route
-	if err := routev1.AddToScheme(mgr.GetScheme()); err != nil {
+	if err := routev1.Install(mgr.GetScheme()); err != nil {
```
Comment (rigazilla) [inline pkg/launcher/launcher.go:121]: just curious, why `Install` instead of `AddToScheme`?

Comment (dmvolod) [reply pkg/launcher/launcher.go:121]: `AddToScheme` marked as deprecated, but `Install = schemeBuilder.AddToScheme`

**Review by rigazilla — APPROVED** [review]
lgtm!

**Comment by ryanemerson** [comment]
Thanks @dmvolod

**Pull request merged**